### PR TITLE
Added clang-format and formatted

### PR DESCRIPTION
**Pull request by Emersont1**
This clang format does its best to recreate your style, but i thought it
would be best to have the code beautified so it's easy to bring code to
be inline with it

### CODE REVIEW

File: src/KW_button.h [hunk 1]
```diff
@@ -28,46 +28,51 @@
...
 
-
 /**
  * \brief   Creates a button widget.
- * \details The button widget is a composite widget. It uses the second 3x3 tile block (next to the frame block) to render its lines.
- * \param   gui The KW_GUI instance that will hold this button.
- * \param   parent The parent widget of this button.
- * \param   label The label widget to print inside the button.
+ * \details The button widget is a composite widget. It uses the second 3x3 tile
+ * block (next to the frame block) to render its lines. \param   gui The KW_GUI
```
Comment (mobius3) [inline src/KW_button.h:42]: This looks broken, each `\param` should be on the same line

**Review by mobius3 — CHANGES_REQUESTED** [review]
Overall I like the result, but something broke the `\param` documentation, merging it with `\details`. This needs to be fixed. Also, if we could keep the spacing between the cast and the variable/function it looks better to me (`(int) variable` as opposed to `(int)variable`)

File: src/KW_button.h (hunk 1 above)
Comment (Emersont1) [reply src/KW_button.h:42]: that looks like a relatively easy fix with `CommentPragmas:  '^\\.+'`, so I'll add and try that, i just need to commit & PR my radiobutton code

**Review by mobius3 — APPROVED** [review]
Thanks!

**Comment by mobius3** [comment]
I merged the checkbox before merging beautifier (and radio) and now they are conflicting. I'm sorry I should've asked about merging order. Do you care to merge them and update the pull requests?

**Comment by Emersont1** [comment]
sorry, my bad, they should both go in fine now

**Pull request merged**